### DISCUSSION TITLE
Drag and drop image just leaves the local image path, not an img link

### DISCUSSION THREAD

**Discussion by tomchiverton** [community/community#65043] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

This is what happens when I drag an image from Dolphin onto an edit field in FireFox in Ubuntu 23.04

`file:///tmp/Screenshot_20230826_171847.png`

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by arafajs** [comment]
This happens because Firefox doesn't have access to read the local file system directly for security reasons. It only inserts the file path as plain text.

**Reply by tomchiverton** [reply, marked as answer]
I can upload the same one to https://imgbb.com/ fine. If I copy the image from /tmp to /home/$USER I can upload to GitHub fine too.

Looks like yet another problem with FireFox Snap
